src/views/Home/Article/ArticleDetail/index.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { useCallback } from 'react';
+
+import { isDefined } from '@togglecorp/fujs';
+import {
+    IoClose,
+    IoCheckmarkSharp,
+} from 'react-icons/io5';
+
+import {
+    Modal,
+    Button,
+} from '@togglecorp/toggle-ui';
+
+import TagInput from '#components/TagInput';
+
+import { useRequest } from '#utils/request';
+import { MultiResponse } from '#utils/common';
+
+import styles from './styles.css';
+
+interface ArticleDetail {
+    title: string;
+    url: string;
+    articleId: string;
+    content: string;
+    date: string;
+    language?: string;
+    locations?: string[];
+    regions?: string[];
+    tags?: string[];
+    themes?: string[];
+    searchScore: number;
+}
+
+interface Props {
+    onModalClose: () => void;
+    articleId: string;
+}
+
+function ArticleDetail(props: Props) {
+    const {
+        onModalClose,
+        articleId,
+    } = props;
+
+    const {
+        response,
+    } = useRequest<MultiResponse<ArticleDetail>>({
+        url: `/api/articles/${articleId}`,
```

Comment:
```suggestion
        url: `/api/articles/${articleId}/`,
```
Always end your URLs with a `/`. Not having `/` at the end with redirect it to API with `/` and if servers are not configured properly, requests will be different.
Bottom line, adding a `/` at the end will reduce complications in most of the cases.